package.json

```diff
@@ -92,8 +94,9 @@
     "ganache-cli": "6.9.0",
     "jest": "^24.1.0",
     "mocha-junit-reporter": "^1.18.0",
-    "solidity-coverage": "0.6.7",
-    "truffle": "^5.0.34",
+    "ora": "^4.0.3",
+    "solidity-coverage": "^0.7.1",
+    "truffle": "^5.1.12",
```

Comment:
Upgraded truffle because it has some bug fixes wrt swallowing errors I ran into. Probably not necessary.

Added `ora` because it looks like an accidentally missing dependency of @truffle/config.